tests/core/inference/test_hugging_face_model.py

```diff
@@ -0,0 +1,758 @@
+import unittest
+from unittest import mock
+
+from pydantic import ValidationError
+
+from marqo.core.inference.inference_models.hugging_face_model_properties import HuggingFaceModelProperties, PoolingMethod
+from marqo.tensor_search.models.external_apis.hf import HfModelLocation
+from marqo.tensor_search.models.private_models import ModelLocation
+from marqo.core.inference.inference_models.hugging_face_model import HuggingFaceModel
+from marqo.s2_inference.errors import InvalidModelPropertiesError
+import numpy as np
+
+
+class TestHuggingFaceModel(unittest.TestCase):
+    """Test initializing the HuggingFaceModel with valid properties."""
+
+    E5_BASE_V2_MODEL_EMBEDDINGS = np.squeeze(
```

Comment:
nit. consider having these fixed embeddings in a separate json file, it's easier to read and maintain.

Reply:
I will fix this after the release